src/components/CopyUrlButton.js

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2018, The Linux Foundation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { Button, Tooltip, OverlayTrigger } from 'react-bootstrap'
+import { CopyToClipboard } from 'react-copy-to-clipboard'
+import FontAwesome from 'react-fontawesome'
+
+export default class CopyUrlButton extends Component {
+
+  static propTypes = {
+    path: PropTypes.string
```

Comment:
should have bsStyle and className as declared props (with defaults ideally)

Reply:
Sounds good, will define and add defaults for all props.